File: app/src/main/java/org/magnum/logger/communication/email/EmailTable.java
package org.magnum.logger.communication.email;

import org.magnum.logger.Database;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;

public class EmailTable extends Database
{
	private static final String TABLE_NAME = "Email";
	private static final String COLUMN_NAMES[] = { "Time", "Address"};// , "Received" };

	public EmailTable(Context context)
	{
		super(context);
	}

	public static void create(SQLiteDatabase db)
	{
		db.execSQL("CREATE TABLE IF NOT EXISTS " + TABLE_NAME + "( Time INTEGER NOT NULL,  Address INTEGER NOT NULL)");//, Received INTEGER NOT NULL )");
	}

	@Override
	public String[] getColumnNames()
	{
		return COLUMN_NAMES;
	}

	@Override
	public String getTableName()
	{
		return TABLE_NAME;
	}
}

File: app/src/main/java/org/magnum/logger/communication/email/EmailTable.kt
package org.magnum.logger.communication.email

import androidx.room.Dao
import androidx.room.Entity
import androidx.room.Insert
import androidx.room.PrimaryKey

const val NAME = "Email"

@Entity(tableName = NAME)
data class EmailRecord(@PrimaryKey val time: Long, val address: String)

@Dao
interface EmailTable {
    @Insert
    fun insert(vararg record: EmailRecord)
}